### PR TITLE
feat(css): update theme

### PR DESCRIPTION
**Pull request by lmssiehdev**
inspired by https://edwardtufte.github.io/tufte-css/

light theme: 

image

dark theme:

image

### CODE REVIEW

File: sass/includes/font.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@font-face {
+  font-family: "et-book";
+  src: url("public/fonts/et-book-roman-line-figures.ttf") format("truetype");
```
Comment (lmssiehdev) [inline sass/includes/font.scss:3]: I added the "public/fonts/et-book-roman-line-figures.ttf" file, but I wasn't able to fully implement it as I'm not familiar with Rust. If you would like to implement the functionality yourself or have any other suggestions, please let me know. Alternatively, you can link to the font hosted on the font's repository:
```css
src: url("https://edwardtufte.github.io/et-book/et-book/et-book-roman-line-figures/et-book-roman-line-figures.ttf") format("truetype");
```

**Comment by HookedBehemoth** [comment]
I'm strongly opposed to website fonts. Maybe serif is a better choice for articles but then users should set the font they prefer in their browser settings.

**Comment by lmssiehdev** [comment]
Would you be open to merging the theme without the font? It still provides a better reading experience than the current one imo.

**Comment by lmssiehdev** [comment]
Okay, here is the final result using the web-safe serif font `Georgia`. Please let me know if you have any further suggestions or feedback.

light theme:

image

dark theme:

image

**Pull request closed without merging**